sdk/eventhub/event-hubs/src/eventProcessor.ts

```diff
@@ -89,11 +103,33 @@ export class EventProcessor {
    *
    * @return {Promise<void>}
    */
-  async start(): Promise<void> {}
+  async start(): Promise<void> {
+    const partitionIds = await this._eventHubClient.getPartitionIds();
+    const partitionContext: PartitionContext = {
+      partitionId: partitionIds[0],
+      consumerGroupName: this._consumerGroupName,
+      eventHubName: this._eventHubClient.eventHubName
+    };
+    const partitionProcessor = this._partitionProcessorFactory(
+      partitionContext,
+      "checkpointManager" as any
```

Comment:
We actually have a `CheckpointManager` class already:
https://github.com/Azure/azure-sdk-for-js/blob/8aad5bec6a003011e86f0c12757f595c0a46c97a/sdk/eventhub/event-hubs/src/checkpointManager.ts#L23

You can new it up and pass it in here. It's ok that it doesn't do anything yet, that'll be tackled in another PR.